docs/developers_guide/organization.rst

```diff
@@ -854,6 +854,13 @@ Some attributes are available after calling the base class' constructor
     Whether to get all of the outputs for the step from the database of
     cached outputs for the MPAS core that this step belongs to
 
+``self.run_as_subprocess``
```

Comment:
@xylar , can you elaborate on how running as a subprocess or not changes the logging behavior?  Is subprocess stdout lost, sent to the terminal/job log, or something else?

Reply:
@matthewhoffman Do you want me to elaborate on that here or in the documentation?